assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/index.js

```diff
@@ -83,6 +83,32 @@ const SearchFunnelWidget = ( {
 		} )
 	);
 
+	const analyticsGoalsData = useInViewSelect( ( select ) =>
+		select( MODULES_ANALYTICS ).getGoals()
+	);
+
+	const analyticsGoalsLoading = useSelect( ( select ) => {
+		if ( ! isAnalyticsConnected ) {
+			return false;
+		}
+
+		return ! select( MODULES_ANALYTICS ).hasFinishedResolution(
+			'getGoals',
+			[]
+		);
+	} );
```

Comment:
Let's use the ternary operator, it will make this hook more concise.
```js
const analyticsGoalsLoading = useSelect( ( select ) => (
	isAnalyticsConnected 
		? ! select( MODULES_ANALYTICS ).hasFinishedResolution( 'getGoals', [] )
		: false;
) );
```

Reply:
Thanks @eugene-manuilov I thought of using the ternary operator. However, the other places where using this style so I used it. I will change it using the ternary operator 👍